### ISSUE TITLE
performance highly degraded when model.eval() in test phase

### ISSUE THREAD

**Issue by PH8411** [magicleap/SuperGluePretrainedNetwork#77]
I train the superglue on my own dataset(some hard examples for matching task ), superpoint for feature extraction and description. 
1. in the traning step, I execute model.train()  for loss backward and model.eval() for validtion, from the validation result , the matching performance is good.
2. however, after training, I load the model weights , and execute model.eval() on my data(testingand training data), the performance highly degraded, but the when I execute model.train() on my data, the results become better again.

Do you have any idea about this or some suggestion about this problem? Looking forward to your reply,thanks.

**Comment by PH8411** [comment]
I remove the BatchNormalization layer, and solve the problem. May be batchsize=1 cause the problem?
